client/stylesheets/_toCat.scss

```diff
@@ -2774,6 +2774,10 @@ i:hover {
     margin: 11px 2px 7px 2px !important;
 }
 
+.headerNameOptions a:hover, .headerNameOptions a:active, .headerNameOptions a:visited, .headerNameOptions a:link {
```

Comment:
Selector `headerNameOptions` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line